phpunit/class-wp-get-block-css-selectors-test.php

```diff
@@ -1,292 +0,0 @@
-<?php
-/**
- * Tests_Get_Block_CSS_Selector class
- *
- * @package WordPress
- */
-
-class WP_Get_Block_CSS_Selector_Test extends WP_UnitTestCase {
```

Comment:
This function is not in Gutenberg anymore. cc @aaronrobertshaw   for confirmation. Let's also confirm that the tests were backported properly.

Reply:
Thanks for the ping 👍 

I can confirm `wp_get_block_css_selector` is now only [defined in core](https://github.com/ramonjd/wordpress-develop/blob/trunk/src/wp-includes/global-styles-and-settings.php#L505). This test was also backported in https://github.com/WordPress/wordpress-develop/pull/4655 and the corresponding file in core still matches the test file being removed in this PR.